static/css/admin.scss

```diff
@@ -64,42 +68,61 @@
         td {
           font-family: $sans;
           color: $main;
+          vertical-align: center;
         }
 
-        td[value="Timestamp"] {
+        td[class="Timestamp"] {
           width: 11.5em;
         }
 
-        td[value="Submitter"] {
-          min-width: 10em;
+        td[class="Submitter"] {
+          min-width: 8em;
         }
 
-        td[value="Status"] {
-          width: 8em;
+        td[class="Status"] {
```

Comment:
This is fine for now, but take another look at what you typed. If we know it's a `<td>` with class of `.status`, then how else could you write it (more correctly)?